packages/core/src/components/tooltip/tooltip.scss

```
@@ -0,0 +1,79 @@
+.atom-tooltip {
```

(block) is important add `scss-utils` in beginning of file

```suggestion
@import '~@atomium/scss-utils/index';

.atom-tooltip {
```